src/routes/+layout.svelte

```diff
@@ -22,6 +22,33 @@
 	});
 </script>
 
+<svelte:head>
+	<!-- HTML Meta Tags -->
+	<title>Swiss Cloud Native Day 2023</title>
+	<meta name="description" content="" />
+
+	<!-- Facebook Meta Tags -->
+	<meta property="og:url" content="https://cloudnativeday.ch" />
+	<meta property="og:type" content="website" />
+	<meta property="og:title" content="Swiss Cloud Native Day 2023" />
+	<meta
+		property="og:description"
+		content="On September 20 & 21, 2023, the local Cloud Native community will organize the third edition of the Swiss Cloud Native Day on Mount Gurten, Bern, Switzerland."
+	/>
+	<meta property="og:image" content="https://cloudnativeday.ch/logo.jpg" />
+
+	<!-- Twitter Meta Tags -->
+	<meta name="twitter:card" content="summary_large_image" />
+	<meta property="twitter:domain" content="cloudnativeday.ch" />
+	<meta property="twitter:url" content="https://cloudnativeday.ch" />
+	<meta name="twitter:title" content="Swiss Cloud Native Day 2023" />
+	<meta
+		name="twitter:description"
+		content="On September 20 & 21, 2023, the local Cloud Native community will organize the third edition of the Swiss Cloud Native Day on Mount Gurten, Bern, Switzerland."
+	/>
+	<meta name="twitter:image" content="https://cloudnativeday.ch/logo.jpg" />
```

Comment:
Is there a benefit to this being served from `static`? Otherwise, I would put it in `src/lib/images` so that it's processed and optimized by the build. Then we could just use the SVG of the logo we already have. If it needs to be a bitmap, we could use a PNG or even a WEBP image.

Reply:
The reason I did it like this was, that I didn't know how to reference the dynamic, cache-busting URLs here, under which the processed images are hosted. If there is an easy way to reference such a dynamic URL, then yes, I agree. We would still need to use jpg, png or webp though, AFAIK. I think SVGs are not allowed here.

Reply:
I think, it can just be `import logo from '$lib/images/logo.webp'; and then `content={logo}`, no?

Reply:
No, I think `content` needs to be a URL in this case.

Reply:
Ok, made it a 1200 x 630 webp file. Hope I got that right :-)